installing/installing_aws/installing-restricted-networks-aws.adoc

```diff
@@ -67,6 +71,10 @@ include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
 
```

Comment:
While not included in these changes (at least not that I could find), the installation-about-restricted-network module states that such an installation can only be performed using user-provisioned installation. Is that true? Why is that a restriction?

Reply:
Thank you for highlighting this. The first paragraph in that module states that a restricted network is one that does not require an active connection to the Internet to obtain software components. I can't see any other references to "restricted" other than in the context of "restricted network" in that module.

1. Is it possible to do an IPI installation on a network that does not have an active connection to the internet, but doe still have access to cloud platform APIs?

2. Is it possible to do an IPI installation from a mirrored registry?

Reply:
Yes, that is possible.

Reply:
@staebler thank you for highlighting this.

After reviewing`modules/installation-about-restricted-network.adoc` in detail, I think that your point should be addressed in a dedicated pull request that focuses on updating that module accordingly. I will address that in a future PR and will link it back to this comment once I have created it.